static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,191 @@
+      <Button
+        size="sm"
+        priority="primary"
+        // TODO(cathy): open up invite modal
+        data-test-id="view-all-missing-members"
+      >
+        {t('View All')}
+      </Button>
+    </MemberCard>
+  );
+
+  return (
+    <StyledCard data-test-id="invite-banner">
+      <CardTitleContainer>
+        <CardTitleContent>
+          <CardTitle>{t('Bring your full GitHub team on board in Sentry')}</CardTitle>
+          <Subtitle>
+            {tct('[missingMemberCount] missing members that are active in your GitHub', {
+              missingMemberCount: missingMembers?.users.length,
+            })}
+            <Tooltip title="Based on the last 30 days of commit data">
+              <IconInfo size="xs" />
+            </Tooltip>
+          </Subtitle>
+        </CardTitleContent>
+        <ButtonContainer>
+          <Button
+            priority="primary"
+            size="xs"
+            // TODO(cathy): open up invite modal
+            data-test-id="view-all-missing-members"
+          >
+            {t('View All')}
+          </Button>
+        </ButtonContainer>
+      </CardTitleContainer>
+      <MemberCardsContainer>{cards}</MemberCardsContainer>
+    </StyledCard>
+  );
+}
+
+export default InviteBanner;
+
+const StyledCard = styled(Card)`
+  padding: ${space(2)};
+  display: flex;
+  overflow: hidden;
+`;
+
+const CardTitleContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const CardTitleContent = styled('div')`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CardTitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeLarge};
+  font-weight: bold;
+  color: ${p => p.theme.gray400};
+`;
+
+const Subtitle = styled('div')`
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 400;
+  color: ${p => p.theme.gray300};
+  display: flex;
+  align-items: center;
+  & > *:first-child {
+    margin-left: ${space(0.5)};
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const ButtonContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: column;
+  grid-column-gap: ${space(1)};
+`;
+
+const MemberCard = styled(Card)`
+  padding: ${space(2)} 18px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: ${space(1)} ${space(0.5)} 0 0;
+  min-width: 330px;
+  justify-content: space-between;
+`;
+
+const MemberCardsContainer = styled('div')`
+  display: flex;
+  overflow-x: scroll;
+`;
+
+const MemberCardContent = styled('div')`
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+`;
+
+const MemberCardContentRow = styled('div')`
+  display: flex;
+  align-items: center;
+  font-size: ${p => p.theme.fontSizeSmall};
+  & > *:first-child {
+    margin-right: ${space(0.75)};
+  }
+  margin-bottom: ${space(0.25)};
+`;
+
+const StyledExternalLink = styled(ExternalLink)`
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+const SeeMoreContainer = styled('div')`
+  font-size: ${p => p.theme.fontSizeLarge};
+`;
```

Comment:
I highly recommend moving this to sentry reusable components and update the code to make it not specific to invite. Like build the carousel as its own component with no data like "CarouselCard" and "Carousel" then reuse here with data. In general it's not good practice to have all logic and UI in one place.

Reply:
okay i will do that. i haven't yet implemented the carousel but i agree it should be made very generic

Reply:
what do you mean when you say the "carousel", like a specific button that moves you along rather than the scrollbar? Also if this is being moved into its own component, would it be easier to do that now or are you waiting until later deliberately?

Reply:
yes to the carousel question. the carousel will be its own component and you will pass the children (in this case the "cards") into it

this PR is already 400 lines long so i will be doing it in a subsequent PR

Reply:
$wag